Signal/src/view controllers/MessagesViewController.m

```diff
@@ -1365,9 +1389,26 @@ - (void)handleWarningTap:(TSInteraction *)interaction
     }
 }
 
-
 - (void)moviePlayBackDidFinish:(id)sender {
-    DDLogDebug(@"playback finished");
+    DDLogDebug(@"%@ %s", self.tag, __PRETTY_FUNCTION__);
+}
+
+- (void)moviePlayerWillExitFullscreen:(id)sender {
```

Comment:
I would have expected doing the same thing on *will*Exit and *did*Exit to be redundant. Can you leave a comment to explain how you arrived at this?